walkthroughs/eks/getting-started-with-appmesh/infrastructure/appmesh_templates/appmesh-virtual-router-appserver-v2.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualRouter
+metadata:
+  namespace: yelb
+  name: yelb-appserver
+spec:
+  awsName: yelb-appserver-virtual-router
+  listeners:
+    - portMapping:
+        port: 4567
+        protocol: http
+  routes:
+    - name: route-to-yelb-appserver
+      httpRoute:
+        match:
+          prefix: /
+        action:
+          weightedTargets:
+            - virtualNodeRef: 
+                name: yelb-appserver-v2
+              weight: 1
+        retryPolicy:
+            maxRetries: 2
+            perRetryTimeout:
+                unit: ms
+                value: 2000
+            httpRetryEvents:
+                - server-error
+                - client-error
```

Comment:
Is `client-error` required? These events usually are not retryable in customer architectures (but we include the option just in case). Same for other retry policies in this PR.

Reply:
Implementing retries because of our best practices: https://docs.aws.amazon.com/app-mesh/latest/userguide/best-practices.html